content/docs/user-guide/project-structure/dvcyaml-files.md

```diff
@@ -457,6 +462,32 @@ validation and auto-completion.
 > See also
 > [How to Merge Conflicts](/doc/user-guide/how-to/merge-conflicts#dvcyaml).
 
+### Stage commands
```

Comment:
command should go before metrics plots outputs, etc? we should be describing things in their order. In every section we can put a small snippet again to get some context back what part of dvc.yaml we are talking about. 

Section names could be a bit more formal: "`cmd`: stage command"

Reply:
I moved **Stage commands** inside the **Stages** section, makes more sense to have it there indeed.

> In every section we can put a small snippet again
Section names could be a bit more formal

Good ideas. I'll create a follow up ticket for that reference not to address in this PR.